tests/run/sequential_parallel.pyx

```diff
@@ -378,10 +378,6 @@ def test_prange_continue():
 
 def test_nested_break_continue():
     """
-    DISABLED. For some reason this fails intermittently on jenkins, with
-    the first line of output being '0 0 0 0'. The generated code looks
-    awfully correct though... needs investigation
-
     >> test_nested_break_continue()
```

Comment:
Actually, it's this line that disabled the test (I had to take a second look, too). Note the `>>` at the beginning, which isn't Python's `>>>`.